.github/workflows/rust.yml

```diff
@@ -44,3 +48,18 @@ jobs:
         run: cargo test --all-targets --all-features
       - name: Run clippy check
         run: cargo clippy --all-targets --all-features -- -D warnings
+  nix_build:
+    name: Nix build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: cachix/install-nix-action@v27
+      - name: Check Nix flake
+        run: nix flake check --all-systems
```

Comment:
This didn't work, the nixpkgs-mozilla was only able to run on Linux as far as I could tell

Reply:
I don't think that we want to try to do a nix package at every PRs.
I suggest to do a new workflow like for snap packages https://github.com/amber-lang/amber/blob/master/.github/workflows/snap.yml that is executed only on push on master

Reply:
Understood. That does mean that the flake.nix can be broken on `master`. But it does at least mean that the maintainers will know it's broken and if I have a problem using the flake I'll be able to come and see if it's just me. So it's still a good step forwards :)